CRITemplateModule.asn

```diff
@@ -76,7 +76,9 @@ SubjectPublicKeyInfoTemplate{PUBLIC-KEY:IOSet} ::= SEQUENCE {
 }
 
 
-id-extensionReqTemplate AttributeType ::= {pkcs-9 TBD3}
+id-extensionReqTemplate AttributeType ::=
```

Comment:
```suggestion
id-aa-extensionReqTemplate AttributeType ::=
```
As suggested by @russhousley.

Reply:
It needs to change in the ATTRIBUTE definition to match too.